ProductivityFeatures/Update168.cs

```diff
@@ -2,6 +2,8 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Text;
+using System.IO;
```

Comment:
Seems you double referenced `System.IO`. Don't know if this is also an example for VS.